ert3/storage/_storage.py

```diff
@@ -395,6 +400,21 @@ def get_experiment_parameters(
     return list(response.json())
 
 
+def get_experiment_responses(*, workspace: Path, experiment_name: str) -> Iterable[str]:
+
+    experiment = _get_experiment_by_name(experiment_name)
+    if experiment is None:
+        raise ert3.exceptions.NonExistantExperiment(
+            f"Cannot get responses from non-existing experiment: {experiment_name}"
+        )
+
+    ensemble_id = experiment["ensembles"][0]  # currently just one ens per exp
+    response = requests.get(url=f"{_STORAGE_URL}/ensembles/{ensemble_id}/responses")
```

Comment:
Should not this also use the `_get_from_server`?

Reply:
:see_no_evil: :+1: